submissions/yaripey/dom-task/js/main.js

```diff
@@ -83,6 +73,5 @@ document.addEventListener('DOMContentLoaded', () => {
     document.querySelector('aside').classList.toggle('toggled')
   })
 
-  // This imitates click for the content on page when first opened.
   document.querySelector('#about').click()
```

Comment:
I dont recommend using click emulation. Better to make the first item active by default.

Reply:
After shifting responsibilities in functions, now it was easy to render the first page right away, without `click`, thanks.